catroid/src/androidTest/java/org/catrobat/catroid/test/physics/look/PhysicsLookPositionAndAngleTest.java

```diff
@@ -57,6 +57,7 @@ public void setUp() throws Exception {
 		physicsObject = mock(PhysicsObject.class);
 		physicsWorldSpy = Mockito.spy(new PhysicsWorld(1920, 1600));
 		when(physicsWorldSpy.getPhysicsObject(sprite)).thenReturn(physicsObject);
+		Mockito.doNothing().when(physicsObject).setFixedRotation(Mockito.anyBoolean());
```

Comment:
please explain :D

Reply:
@juliajulie95 The test failed before due to a nullpointer error. The statement technically makes the method `setFixedRotation`of the mocked `physicsObject` (regardless of the parameter value) act as if it has been successfully executed. It's functionality is irrelevant to the test anyway.